arithmaticOperation.js

```diff
@@ -0,0 +1,23 @@
+/**
+*   Calculate the area of a rectangle.
+*
+*   length: The length of the rectangle.
+*   width: The width of the rectangle.
+*   
+*	Return a number denoting the rectangle's area.
+**/
+function getArea(length, width) {
+    return length*width;
```

Comment:
add a check if length and width are undefined or not. because you code will break if you pass undefined width , length

Reply:
Ok will do it